Screenbox/Pages/PlayerPage.xaml

```diff
@@ -259,9 +259,11 @@
 
                 <TextBlock
                     x:Name="SubtitleText"
+                    Foreground="{StaticResource TextFillColorSecondaryBrush}"
                     OpticalMarginAlignment="TrimSideBearings"
+                    Style="{StaticResource BodyTextBlockStyle}"
                     Text="{x:Bind ViewModel.Media.AltCaption, Mode=OneWay, FallbackValue={}}"
-                    TextTrimming="CharacterEllipsis"
+                    TextWrapping="NoWrap"
```

Comment:
Is there any reason why only the subtitle text has text trimming and not the title text?

Reply:
Both don't need it, doesn't this commit removes it? I'm confused sorry.